round_I/study-assesment-round-I-426.md

```diff
@@ -0,0 +1,57 @@
+# STUDY ASSESSMENT - ROUND I
+
+Please replace '???' (triple question mark) in form below with respective answers. 
+
+## Meta information
+
+|                                               |                                                                                                        |
+| ---                                           |--------------------------------------------------------------------------------------------------------|
+| 1. Assessor name                              | `Lev Bagryansky`                                                                                       |
+| 2. Assessment date (`DD.MM.YYY`)              | `09.06.2023`                                                                                           | 
+| 3. Study title                                | `KLEESPECTRE : Detecting Information Leakage through Speculative Cache Attacks via Symbolic Execution` |
+| 4. Publication year in format YYYY, e.g. 2023 | `2020`                                                                                                 |
+  
+## Has the study raised any issues of Symbolic Execution?
+
+Please examine Abstract, Introduction and Contribution sections on the subject of mentioning issues/difficulties/challenges/etc. of 
+Symbolic Execution.
+  
+5. (Yes/No): `No`
+
+If you answered 'Yes' please provide references to the location(s) within the study supporting your answer:  
+(add as many list items 5.x as required in format `page No., citation`. Remove unnessesary items)
+
+## Has the study introduced any new technique within Symbolic Execution field?
+
+Please examine Abstract, Introduction and Contribution sections on the subject of mentioning any technique/approach/method related to Symbolic Execution.
+  
+6. (Yes/No): `Yes`
```

Comment:
@levBagryansky Do suggested techniques address any problem challenge of the area?

Reply:
@mximp it allows to use symbolic execution with considering the CPU cache.